Gaurav.java

```diff
@@ -11,7 +11,7 @@ public static void main(String[] args) {
 				System.out.println("This is Gaurav Garg testing using GIT");

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS106)